manifests/c/Cube/CubePlatform/3.7.8.1/Cube.CubePlatform.installer.yaml

```diff
@@ -1,71 +1,67 @@
-# Created with YamlCreate.ps1 v2.2.6 $debug=QUSU.CRLF.7-3-4.Win32NT
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
-
-PackageIdentifier: Cube.CubePlatform
-PackageVersion: 3.7.8.1
-InstallerType: exe # Advanced Installer
-Scope: machine
-InstallModes:
-- interactive
-- silent
-- silentWithProgress
-InstallerSwitches:
-  Silent: /exenoui /quiet /norestart
-  SilentWithProgress: /exenoui /passive /norestart
-  InstallLocation: APPDIR="<INSTALLPATH>"
-  Log: /log "<LOGPATH>"
-ExpectedReturnCodes:
-- InstallerReturnCode: -1
-  ReturnResponse: cancelledByUser
-- InstallerReturnCode: 1
-  ReturnResponse: invalidParameter
-- InstallerReturnCode: 87
-  ReturnResponse: invalidParameter
-- InstallerReturnCode: 1601
-  ReturnResponse: contactSupport
-- InstallerReturnCode: 1602
-  ReturnResponse: cancelledByUser
-- InstallerReturnCode: 1618
-  ReturnResponse: installInProgress
-- InstallerReturnCode: 1623
-  ReturnResponse: systemNotSupported
-- InstallerReturnCode: 1625
-  ReturnResponse: blockedByPolicy
-- InstallerReturnCode: 1628
-  ReturnResponse: invalidParameter
-- InstallerReturnCode: 1633
-  ReturnResponse: systemNotSupported
-- InstallerReturnCode: 1638
-  ReturnResponse: alreadyInstalled
-- InstallerReturnCode: 1639
-  ReturnResponse: invalidParameter
-- InstallerReturnCode: 1641
-  ReturnResponse: rebootInitiated
-- InstallerReturnCode: 1640
-  ReturnResponse: blockedByPolicy
-- InstallerReturnCode: 1643
-  ReturnResponse: blockedByPolicy
-- InstallerReturnCode: 1644
-  ReturnResponse: blockedByPolicy
-- InstallerReturnCode: 1649
-  ReturnResponse: blockedByPolicy
-- InstallerReturnCode: 1650
-  ReturnResponse: invalidParameter
-- InstallerReturnCode: 1654
-  ReturnResponse: systemNotSupported
-- InstallerReturnCode: 3010
-  ReturnResponse: rebootRequiredToFinish
-UpgradeBehavior: install
-Protocols:
-- asuka
-ReleaseDate: 2023-05-25
-Installers:
-- Architecture: x64
-  InstallerUrl: https://download.cubejoy.com/app/3.7.8.1/CubeSetup_v3.7.8.1.exe
-  InstallerSha256: 4752BDA4E422E8CB8BD7760724D30B3EA8493F2AA734032FA33AA87D426D547B
-- InstallerLocale: zh-HK
-  Architecture: x64
-  InstallerUrl: https://download.cubejoy.com/app/3.7.8.1/CubeSetup_HK_TC_v3.7.8.1.exe
-  InstallerSha256: 8D3DFFF14B6DD7C443F156CC77529DC07C85CD926F610B13C9BFC8FB1DDAF0F8
-ManifestType: installer
-ManifestVersion: 1.4.0
+# Created with [WinGet Automation](https://github.com/vedantmgoyal2009/winget-manifests-manager) using Komac v1.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Cube.CubePlatform
+PackageVersion: 3.7.8.1
+InstallerLocale: zh-HK

```

Comment:
The installer-wise property was moved to manifest-wide unexpectedly, causing the two installer entries "duplicated" here.
@russellbanks @vedantmgoyal2009